### PR TITLE
[node] Casting quickfix in app install

### PR DESCRIPTION
**Pull request by LayneHaber**
### Description

There is a weird issue where the parameters are not being properly casted as a BN on the counterparty's deposit. This is an inelegant, but effective, fix where the parameters are always casted as they are parsed from the context.

### Related issues

Addresses issue #1949 

- [ ] Deploy preview is functional

### CODE REVIEW

File: packages/node/src/protocol/install.ts [hunk 1]
```diff
@@ -334,10 +334,10 @@ function computeStateChannelTransition(
   if (initiatorDepositTokenAddress !== responderDepositTokenAddress) {
     tokenIndexedBalanceDecrement = {
       [initiatorDepositTokenAddress]: {
-        [initiatorFbAddress]: initiatorBalanceDecrement
+        [initiatorFbAddress]: bigNumberify(initiatorBalanceDecrement)
```
Comment (cf19drofxots) [inline packages/node/src/protocol/install.ts]: Do we need to recast again here and below if we've done it in the context of this call above already?

**Comment by snario** [comment]
Oh I see the issue.

The `params` aren't getting serialized.

It is the `params` variable here that needs to be serialized:

https://github.com/counterfactual/monorepo/blob/master/packages/node/src/message-handling/handle-protocol-message.ts#L39

I would write a generic serializer that detects `_hex` and serialize at this point.

There is similar code to copy here: https://github.com/counterfactual/monorepo/blob/master/packages/node/src/models/app-instance.ts#L108

**Comment by cf19drofxots** [comment]
@rhlsthrm bump the node version (and its references in consumer packages) and we can merge this

**Comment by rhlsthrm** [comment]
Bumped versions and changed fix to be more general.

**Pull request merged**